File: app/src/main/java/com/iamagamedev/mybrandnewgame/levels/LevelBattle.java
package com.iamagamedev.mybrandnewgame.levels;

import java.util.ArrayList;

/**
 * Created by Михан on 06.06.2017.
 */

public class LevelBattle extends LevelData {

    public LevelBattle(){
        tiles = new ArrayList<>();
        this.tiles.add("..........");
        this.tiles.add("..........");
        this.tiles.add(".p......x.");
        this.tiles.add("wwwwwwwwww");
    }
}

File: app/src/main/java/com/iamagamedev/mybrandnewgame/levels/LevelBattle.kt
package com.iamagamedev.mybrandnewgame.levels

import com.iamagamedev.mybrandnewgame.background.BackgroundData
import java.util.*

/**
 * Created by Михан on 06.06.2017.
 */
class LevelBattle : LevelData() {
    init {
        tiles = ArrayList()
        tiles?.also {
            it.add("                                  ")
            it.add("                                 .")
            it.add("         a                       z")
            it.add("wwwwwwwwwwwwww                   1")
            it.add("wwwwwwwwwwwwww                   s")
            it.add("wwwwwwwwwwwwww                    ")
            it.add("pwwwwwwwwwwwww                    ")
        }

        this.backgroundDataList = arrayListOf()
        this.backgroundDataList?.add(BackgroundData("mountain", true, -1, -3f, 10f, 10f, 15))
        this.backgroundDataList?.add(BackgroundData("forest", true, 1, 20f, 24f, 24f, 4))
    }
}